File: javascript/claude/p27.js
// Task p27: Declare totalSales, add sales for 3 months, print total
function getMonthlySales() {
  /**
   * Returns a random number between 0 and 100000
   */
  return Math.floor(Math.random() * 100001);
}

// Declare totalSales variable
let totalSales = 0;

// Add the sales for each month to the totalSales variable
const month1Sales = getMonthlySales();
totalSales += month1Sales;

const month2Sales = getMonthlySales();
totalSales += month2Sales;

const month3Sales = getMonthlySales();
totalSales += month3Sales;

// Print the total sales to the screen
console.log(`Month 1: $${month1Sales}`);
console.log(`Month 2: $${month2Sales}`);
console.log(`Month 3: $${month3Sales}`);
console.log(`Total Sales for Q1: $${totalSales}`);

File: c/claude/p27.c
// Task p27: Declare totalSales, add sales for 3 months, print total
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int getMonthlySales()
{
    /**
     * Returns a random number between 0 and 100000
     */
    return rand() % 100001;
}

int main()
{
    srand(time(NULL));

    // Declare totalSales variable
    int totalSales = 0;

    // Add the sales for each month to the totalSales variable
    int month1Sales = getMonthlySales();
    totalSales += month1Sales;

    int month2Sales = getMonthlySales();
    totalSales += month2Sales;

    int month3Sales = getMonthlySales();
    totalSales += month3Sales;

    // Print the total sales to the screen
    printf("Month 1: $%d\n", month1Sales);
    printf("Month 2: $%d\n", month2Sales);
    printf("Month 3: $%d\n", month3Sales);
    printf("Total Sales for Q1: $%d\n", totalSales);

    return 0;
}
